bundles/org.openhab.binding.ipcamera/src/main/java/org/openhab/binding/ipcamera/internal/DahuaHandler.java

```diff
@@ -352,6 +437,16 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                             .sendHttpGET("/cgi-bin/configManager.cgi?action=setConfig&MotionDetect[0].Enable=false");
                 }
                 return;
+            case CHANNEL_MOTION_DETECTION_LEVEL:
+                int motionlevel = Math.round(Float.valueOf(command.toString()));
```

Comment:
```suggestion
if (command instanceof PercentType) {
int motionlevel = ((PercentType) command).toBigDecimal().intValue();
}                
```
More efficient and should prevent the ON and OFF commands from causing issues.